ui-tests-protractor/e2e/integrations/integrations-sf-db.feature

```diff
@@ -65,3 +64,5 @@ https://drive.google.com/file/d/0B_udTBpEdqO8WUxzTWVKX1NsME0/view
         # assert integration is present in list
         Then Camilla is presented with "Salesforce to PostresDB E2E" integration details
         And she clicks on the "Done" button
+        # wait for integration to get in active state
+        Then she wait until integration "Twitter to Salesforce E2E" get into "Active" state
```

Comment:
I have a feeling there's different integration name for this scenario :)

Reply:
Copy paste error :-)